src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Utf8Parser/Utf8Parser.Integer.Unsigned.N.cs

```diff
@@ -11,7 +11,7 @@ public static partial class Utf8Parser
     {
         private static bool TryParseByteN(ReadOnlySpan<byte> source, out byte value, out int bytesConsumed)
         {
-            if (source.Length < 1)
+            if (source.IsEmpty)
```

Comment:
It'd be great to pick one or two examples and validate that the assembly the JIT generates after this change is at least as good.  It should be, if not better, but we should double-check, as some of these are likely hot-paths.

Reply:
https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIGYACMhgYQYG8aHuGAHKAJYA3bBhhNySBsAgQANgwCyACgBKMbABMA8gDs5ATwDKvbLoA8wA2IB8DXBACuUMDACUXHp2o9fDAQBmDMoOzq4AdACSuACi+LwYBh4+fqnEAOwMAdhyuDAA3J6pRX4ZDBhQjgUlAL4lJfzCouLEktKyCgByaho6+samFla29k4u7iXeqdyBwaHj4QAyMLoA5hgAFgzmDOTJ06WZ2bnVKcVnh+WVp7511DVAA=

For this case, the assembly is the same size.